yb-voyager/src/dbzm/dbzm.go

```diff
@@ -25,26 +25,24 @@ func init() {
 	} else {
 		DEBEZIUM_DIST_DIR = "/opt/yb-voyager/debezium-server"
 	}
-
-	DEBEZIUM_CONF_DIR = filepath.Join(DEBEZIUM_DIST_DIR, "conf")
```

Comment:
remove declaration of `DEBEZIUM_CONF_DIR` variable if it is not used anywhere anymore

Reply:
done